### PR TITLE
ES6 Implementation

### PR DESCRIPTION
**Pull request by danigalindo10**
For this project, I changed the previous awesome books to ES6 syntax following the rules below: ✅ 

- Setup my project using creating new HTML, CSS, and JS files.
- Re-created my awesome books app using ES6 syntax and divided my code into modules.
- Displayed the current date below the navigation bar using Luxon Library.

### CODE REVIEW

**Comment by danigalindo10** [comment]
# 🔴 🔴 🔴 To the reviewer 🔴 🔴 🔴
Hello dear reviewer, I know that I have a webhint error but I don't really know what's the problem. When I go to my browser to see my project, I can see that all actually works well and when I run webhint on my local machine, it doesn't appear any error. Please could you tell me what's the problem with this linter error? I have no idea what I am doing wrong 😢.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @danigalindo10 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors, I saw your comment, the problem seems like you import a statement outside a module as shown in the error report, below is a screenshot.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import Store from './modules/store';
+import awesomeBooks from './modules/awesome-books';
+import Book from './modules/book';
+// import { DateTime } from './node_modules/luxon/src/luxon.js';
+
```
Comment (HAMMAS-SALEEM) [inline index.js]: - [x] Please use luxon for showing the date and time on your page :+1:

File: index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  addNewBookSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+});
+
+// contactLink
+contactLink.addEventListener('click', () => {
+  allBooksSection.classList.add('hide');
+  addNewBookSection.classList.add('hide');
+  contactSection.classList.remove('hide');
+});
```
Comment (HAMMAS-SALEEM) [inline index.js:73]: - [x] You have made a great effort in making this project but your project navigation links are not working. Please fix them :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @danigalindo10 :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following JavaScript best practics
:heavy_check_mark: Following HTML, CSS best practices

Good job on implementing some of the changes requested by the previous review but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @danigalindo10, 

Good job following what the previous reviewers mentioned, your project now is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**